rpc/grpc/proto/blockchain.proto

```diff
@@ -12,29 +12,42 @@ option (gogoproto.sizer_all) = true;
 option (gogoproto.goproto_registration) = true;
 option (gogoproto.messagename_all) = true;
 option (gogoproto.protosizer_all) =false;
-option (gogoproto.testgen_all) = true;
 
 
-// Account Service definition
-service Accounts {
+
+// BlockChain  Service definition
+service BlockChain{
   rpc GetAccount(AddressRequest) returns (AccountResponse);
+  rpc GetAccounts(Empty) returns (AccountsResponse);
   rpc GetStorage(StorageAtRequest) returns (StorageResponse);
   rpc GetStorageAt(StorageAtRequest) returns(StorageResponse);
-  rpc GetAccounts(Empty) returns (AccountsResponse);
   rpc GetValidator(AddressRequest) returns (ValidatorResponse);
   rpc GetValidators(Empty) returns (ValidatorsResponse);
+  rpc Getstatus(Empty) returns(StatusResponse);
+  rpc GetBlock(BlockRequest) returns(BlockResponse);
+  rpc GetBlocks(BlocksRequest) returns (BlocksResponse);
+  rpc GetGenesis(Empty) returns(GenesisResponse);
+  rpc GetChainID(Empty) returns(ChainResponse);
+  rpc GetLatestBlock(BlockRequest) returns(BlockResponse);
+  rpc GetConsensusState(Empty) returns (ConsensusResponse);
+  rpc GetBlockTxs(BlockRequest)returns(BlockTxsResponse);
+
 }
 
 message Empty {
 }
 
 message AccountsResponse{
```

Comment:
Account**s**Response/AccountResponse should be same as Validor**s**Response/ValidatorResponse